test/integration/targets/setup_podman/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+- name: Install podman
+  yum:
+    name: podman
+    state: present
+  when: ansible_facts.pkg_mgr in ['yum', 'dnf']
```

Comment:
This should probably limit based on distro/release because podman isn't available for RHEL/CentOS 6.x and for  RHEL/CentOS 7.x you have to add an extra subscription channel (or repo in CentOS's case).